plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RequestLoginToRepositoryExtensionPoint.java

```diff
@@ -30,7 +30,7 @@
 import org.pentaho.di.repository.Repository;
```

Comment:
Copyright date on this file needs to be updated.

Reply:
fixed